anoop_assignment-workspace

an-00-p opened the pull request:
This is the required step for next round.

ayushsgithub reviewed (changes requested):
What does these lines of code do?

an-00-p commented:
> What does these lines of code do?

Sir 
I am not used to with Github that much , I am still learning and that was my mistake during the submission of assignment. I have corrected it and added a chatbot (Google chat bot API). you can check the changes,

client/chat_bot.py

ayushsgithub commented:
> > What does these lines of code do?
> 
> Sir I am not used to with Github that much , I am still learning and that was my mistake during the submission of assignment. I have corrected it and added a chatbot (Google chat bot API). you can check the changes,
> 
> client/chat_bot.py

Hey bro, we all are learning and it's ok to make mistakes in open source, but I can't find your changes in your pr, please add a screenshot of the file changes and create a new branch for it.
And if you want to know anything about open source, git or GitHub, you can dm me on any platform, I'm not a pro but I can definitely help you.

an-00-p commented:
https://github.com/an-00-p/anoop_assign/tree/chat_bot

On Sat, Sep 2, 2023 at 1:42 PM Ayush ***@***.***> wrote:

> What does these lines of code do?
>
> Sir I am not used to with Github that much , I am still learning and that
> was my mistake during the submission of assignment. I have corrected it and
> added a chatbot (Google chat bot API). you can check the changes,
>
> client/chat_bot.py
>
> Hey bro, we all are learning and it's ok to make mistakes in open source,
> but I can't find your changes in your pr, please add a screenshot of the
> file changes and create a new branch for it.
> And if you want to know anything about open source, git or GitHub, you can
> dm me on any platform, I'm not a pro but I can definitely help you.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/practicehealth/assignemnt/pull/3#issuecomment-1703761052>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A25SX36D23XS3BCZ3HGHGQ3XYLTAFANCNFSM6AAAAAA4HMOFA4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
